pkg/build/admission/admission.go

```diff
@@ -43,21 +43,57 @@ func NewBuildByStrategy(client client.Interface) admission.Interface {
 const (
 	buildsResource       = "builds"
 	buildConfigsResource = "buildconfigs"
+
+	cloneSubresource       = "clone"
```

Comment:
they need a better place to live ... I'll move them. We need to use them wherever we reference these resources like the boostrap policy